pkgs/applications/editors/eclipse/plugins.nix

```diff
@@ -135,20 +197,14 @@ rec {
     };
   };
 
-  antlr-runtime_4_5 = buildEclipsePluginBase rec {
+  antlr-runtime_4_5 = buildEclipsePlugin rec {
```

Comment:
Not sure why these were using buildEclipsePluginBase to begin with, buildEclipsePlugin would always have been simpler. But this is now a forced move, since buildEclipsePluginBase doesn't work any more.